homeassistant/components/zha/number.py

```diff
@@ -1053,3 +1054,74 @@ class MinHeatSetpointLimit(ZCLHeatSetpointLimitEntity):
     _attr_entity_category = EntityCategory.CONFIG
 
     _max_source = Thermostat.AttributeDefs.max_heat_setpoint_limit.name
+
+
+@CONFIG_DIAGNOSTIC_MATCH(
+    cluster_handler_names=CLUSTER_HANDLER_THERMOSTAT,
+    quirk_ids={DANFOSS_ALLY_THERMOSTAT},
+)
+# pylint: disable-next=hass-invalid-inheritance # needs fixing
+class DanfossExerciseTriggerTime(ZHANumberConfigurationEntity):
+    """Danfoss proprietary attribute to set the time to exercise the valve."""
+
+    _unique_id_suffix = "exercise_trigger_time"
+    _attribute_name: str = "exercise_trigger_time"
+    _attr_translation_key: str = "exercise_trigger_time"
+    _attr_native_min_value: int = 0
+    _attr_native_max_value: int = 1439
+    _attr_mode: NumberMode = NumberMode.BOX
+    _attr_native_unit_of_measurement: str = "Minutes"
```

Comment:
This should probably be `UnitOfTime.MINUTES`

Reply:
Done